.env.cluster

```diff
@@ -1,6 +1,6 @@
 # General
 
-STATEFUL_NODES=cluster
+NODE_MODE=cluster
```

Comment:
cool nice idea! fixed in [cd07c0f](https://github.com/jembi/platform/pull/188/commits/cd07c0ff28875419509012d0d94b88a5fd22af90)

Reply:
BTW I don't think This PR should be merged until I finish all the packages, I made changes to the general functions so this may effect the other scripts, so I think for safety reasons, I suggest that this PR will be the branch that all the other PRs will be merged to until all the packages are updated